### PR TITLE
[Docs] - Mozilla/Reps - README.md - rethorics (reviewing grammar)

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,6 @@
 
 ![Participate](https://wiki.mozilla.org/images/thumb/4/47/Remo_logo_vertical.png/150px-Remo_logo_vertical.png)
 
-This repository is used for tracking [issues](https://github.com/mozilla/Reps/issues) directly related to the work of the Mozilla Reps program.
+This repository is used for the purpose of tracking [issues](https://github.com/mozilla/Reps/issues) directly related to the work of the Mozilla Reps program.
```
Comment (MichaelKohler) [inline README.md:9]: Why is this needed?

File: README.md [hunk 2]
```diff
@@ -6,6 +6,6 @@
...
 
-If you'd like to propose new work or get in touch with the program, head over to discourse here: https://discourse.mozilla-community.org/c/reps
+If you'd like to propose new work or get in touch with the program, head over to discourse: https://discourse.mozilla-community.org/c/reps
```
Comment (MichaelKohler) [inline README.md:11]: If we change this, we should also change "discourse" to "Discourse".

**Review by MichaelKohler — CHANGES_REQUESTED** [review]

**Comment by Mte90** [comment]
Seems that this account was deleted so I think that we can close the PR.
